readme.md

```diff
@@ -588,6 +588,9 @@ Noncontiguous selection is also possible.
 You do this in the same way described above for tracks and items.
 That is, press shift+space (OSARA: Enable noncontiguous selection/toggle selection of current chord/note) to switch to noncontiguous selection, move to other chords/notes with shift plus the arrow keys and press shift+space to select/unselect the current chord/note.
 
+#### Note preview in de event list (Windows Only)
```

Comment:
Make level 3 heading, linguistic/capitalisation fix.
```suggestion
### Note Preview in the MIDI Event List (Windows Only)
```

Reply:
I made this heading level 4 initially since it also has to do with the midi editor.

Reply:
I understand. However, the previous section is very specifically about moving through events in the piano roll. It should perhaps say "Piano Roll" instead of "MIDI Editor", but the problem is that this also applies to some other views, everything but the Event List.